### PR TITLE
Building Full Chamber Configs

### PR DESCRIPTION
**Pull request by cbravo135**

### CODE REVIEW

File: confChamber.py [hunk 1]
```diff
@@ -49,14 +53,37 @@
...
-                if event.vfatCH == 10 : writeVFAT(ohboard, options.gtx, int(event.vfatN), "ContReg3", int(event.trimRange),0)
-    except:
+        for event in inF.scurveFitTree :
+            writeVFAT(ohboard,options.gtx,int(event.vfatN),"VFATChannels.ChanReg%d"%(int(event.vfatCH)),int(event.trimDAC)+32*int(event.mask))
+            writeVFAT(ohboard, options.gtx, int(event.vfatN), "ContReg3", int(event.trimRange),0)
+    except Exception as e:
         print '%s does not seem to exist'%options.filename
+        print e
 
+if options.chConfig != None:
```
Comment (jsturdy) [inline confChamber.py]: You don't have to have ```!= None```, ```if options.chConfig``` will treat a ```None``` type as ```False```

File: macros/anaUltraScurve.py [hunk 2]
```diff
@@ -290,7 +297,40 @@ def overlay_fit(VFAT, CH):
...
     myT.Write()
     outF.Close()
     pass
+
+
+
+
+
+
+
```
Comment (jsturdy) [inline macros/anaUltraScurve.py]: Are these all blank lines?  If so, remove

File: macros/anaUltraThreshold.py [hunk 3]
```diff
@@ -161,15 +161,22 @@
 vt1 = []
 for i in range(0,24):
 #    vt1.append([])
-    for j in range(0, 100):
+    for j in range(0, 250):
```
Comment (jsturdy) [inline macros/anaUltraThreshold.py]: What is the meaning of 250 here (and 251 in line 123)?
Is it related to something hardcoded elsewhere?

**Review by jsturdy — COMMENTED** [review]
Couple of minor things and a question.
Not officially requesting changes until receiving response for clarification

**Review by jsturdy — APPROVED** [review]
Looks fine

File: macros/anaUltraScurve.py (hunk 2 above)
Comment (cbravo135) [reply macros/anaUltraScurve.py]: How much do we really care if there are some blank lines at the bottom of the file?

**Pull request merged**